### PR TITLE
End outgoing call with "No Answer" after 2 minutes

### PR DESCRIPTION
**Pull request by michaelkirk**
Also removed some dead and confusing related code from the CallInitiator refactor.

PTAL @charlesmchen

### CODE REVIEW

File: Signal/src/call/CallService.swift [hunk 1]
```diff
@@ -77,8 +77,8 @@ enum CallError: Error {
     case timeout(description: String)
 }
 
-// FIXME TODO do we need to timeout?
-fileprivate let timeoutSeconds = 60
+// Should be roughly synced with Android client for consistency
+fileprivate let timeoutSeconds = 120
```
Comment (charlesmchen) [inline Signal/src/call/CallService.swift]: Maybe rename to `connectingTimeoutSeconds`?  This doesn't apply once you're connected, right?

File: Signal/src/call/SignalCall.swift [hunk 2]
```diff
@@ -17,6 +17,10 @@ enum CallState: String {
     case remoteBusy // terminal
 }
 
+enum CallDirection {
+    case outgoing, incoming
+}
```
Comment (charlesmchen) [inline Signal/src/call/SignalCall.swift]: Hmm, I wonder if its worth trying to DRY this up with the enum in PCC.  Maybe not.

Comment (michaelkirk) [inline Signal/src/call/SignalCall.swift]: done!

File: Signal/src/call/CallService.swift (hunk 1 above)
Comment (michaelkirk) [inline Signal/src/call/CallService.swift]: done.

**Comment by michaelkirk** [comment]
rebased and merged here: https://github.com/WhisperSystems/Signal-iOS/commit/bf1ed9a277f8f8dafbecb94d90f8f1cdb6e97843

**Pull request closed without merging**